### PR TITLE
minetest: add more info regarding server setting

### PR DESCRIPTION
**Pull request by Yutyo**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by marchersimon — APPROVED** [review]

File: pages/common/minetest.md [hunk 1]
```diff
@@ -8,9 +8,9 @@
 
 `minetest`
 
-- Start minetest in server mode:
+- Start minetest in server mode by hosting the specified world:
 
-`minetest --server`
+`minetest --server --world {{name}}`
```
Comment (navarroaxel) [inline pages/common/minetest.md:13]: this works only with `--server` right? why not add an extra example for a specific world?

File: pages/common/minetest.md [hunk 2]
```diff
@@ -8,9 +8,9 @@
 
 `minetest`
 
-- Start minetest in server mode:
+- Start minetest in server mode by hosting the specified world:
```
Comment (navarroaxel) [inline pages/common/minetest.md:11]: Maybe this is better:
```suggestion
- Start minetest in server mode by hosting a specific world:
```

File: pages/common/minetest.md (hunk 1 above)
Comment (Yutyo) [reply pages/common/minetest.md:13]: Yes and `--server` doesn't work at all without specifying a world.

**Comment by Yutyo** [comment]
I merge it then.

**Comment by bl-ue** [comment]
@Yutyo, since @navarroaxel had comments that are now resolved, the conventional thing to do is to re-request his review, whereupon he will either add more comments, or approve (and likely merge it himself.)

However, given that the lines he commented on are the only lines you changed, and you addressed his comments, and there are two approvals, I think it's fine to do what you did.

File: pages/common/minetest.md [hunk 3]
```diff
@@ -8,9 +8,9 @@
 
 `minetest`
 
-- Start minetest in server mode:
+- Start minetest in server mode by hosting a specific world:
 
-`minetest --server`
+`minetest --server --world {{name}}`
```
Comment (sbrl) [inline pages/common/minetest.md:13]: This is a path to a world, not the name of the world. Specifically from the man page:

```
       --world <value>
              Set world path
```

Comment (bl-ue) [reply pages/common/minetest.md:13]: needs a new PR? cc @Yutyo

**Pull request merged**